implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -25,6 +29,18 @@
         this.resolver = resolver;
     }
 
+    private static void buildCompletePropertyFile(File propertyFile) {
```

Comment:
Could you explain how you resolving situation like this: 
1. File exists but someone else write/read to/from this file  

In my option you should create some lock on this file (eg. FileLock.class)